emcc.py

```diff
@@ -1544,6 +1544,10 @@ def check(input_file):
         exit_with_error('wasm2js does not support source maps yet (debug in wasm for now)')
       logger.warning('emcc: JS support in the upstream LLVM+wasm2js path is very experimental currently (best to use fastcomp for asm.js for now)')
 
+    if shared.Settings.BYSYNCIFY:
+      if not shared.Settings.WASM_BACKEND:
```

Comment:
just one line?

Reply:
I was just trying to keep the structure of the other things here: one toplevel if for the "feature", then a body with operations for it. Which do you prefer? I don't feel strongly (the code will be removed with fastcomp anyhow eventually).

Reply:
sgtm